### ISSUE TITLE
Please add the setting to disable automatic check for updates

### ISSUE THREAD

**Issue by iahung** [Dragon-0609/Yuki-Theme#52]
It's really annoying because automatic updates have never worked for me due to permission issues (it can't write to `C:\Program Files (x86)\PascalABC.NET`). I ended up always have to do the update myself by downloading from Github and unpacking to `C:\Program Files (x86)\PascalABC.NET` manually.

**Comment by Dragon-0609** [comment]
Please downgrade the version of the Plugin to [v7](https://github.com/Dragon-0609/Yuki-Theme/releases/tag/v7.0.0). There's a checkbox where you can disable automatic check for updates.

**Comment by Dragon-0609** [comment]
No, by that checkbox you'll no longer receive notification about new version. After rewriting I'm thinking about fixing this bug, which requires admin rights.
